sql/firefox_accounts_derived/fxa_log_device_command_events_v1/query.sql

```diff
@@ -0,0 +1,39 @@
+WITH hmac_key AS (
+  SELECT
+    AEAD.DECRYPT_BYTES(
+      (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.fxa_prod`),
+      ciphertext,
+      CAST(key_id AS BYTES)
+    ) AS value
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.encrypted_keys_v1`
+  WHERE
+    key_id = 'fxa_hmac_prod'
+)
+--
+SELECT
+  `timestamp`,
+  jsonPayload.type,
+  TO_HEX(
+    udf.hmac_sha256((SELECT * FROM hmac_key), CAST(jsonPayload.fields.uid AS BYTES))
+  ) AS user_id,
```

Comment:
Could we please apply the same hmacing logic to the `index`, `target` and `sender` fields? Like `uid`, these fields are unique identifiers stored on the server that could be used to tie metrics back to a specific account.